transmogrifier.js

```diff
@@ -610,6 +613,7 @@ async function transmogrify(manifest) {
     const schemaEntryDatas = [];
     for (schemaEntry of manifest) {
         const entryDatas = [];
+        schemaEntry.schema = schemaEntry.schema ? schemaEntry.schema : "";
```

Comment:
you can use schemaEntry.schema ?? "" like above

Reply:
didn't know you could do that in js too